build.cmd

```diff
@@ -67,7 +73,7 @@ call :build %*
 goto :AfterBuild
 
 :build
-%_buildprefix% msbuild "%_buildproj%" /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%_buildlog%";Append %* %_buildpostfix%
+%_buildprefix% msbuild "%_buildproj%" /p:Configuration=%defaultBuildConfigure% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=diag;LogFile="%_buildlog%";Append %* %_buildpostfix%
```

Comment:
when defaultBuildConfigure is not defined, that means FloatingTestRuntimeDependencies is not defined. It will use the minimum dependencies as usual, nothing unexpected.